components/authz-service/README.md

```diff
@@ -11,39 +11,7 @@ The TOC below is auto-generated by the VSCode "Markdown TOC" plugin; do not edit
 With the plugin installed, this TOC will update automatically when you save the file!
 ]: #
 
-<!-- TOC depthFrom:2 depthTo:3 -->
-
-- [AuthZ Approach](#authz-approach)
-  - [Terminology](#terminology)
-  - [AuthZ with OPA](#authz-with-opa)
-  - [Elements of the AuthZ Query](#elements-of-the-authz-query)
-  - [Elements of the AuthZ Policy](#elements-of-the-authz-policy)
-  - [Mapping Requests to Endpoints to Policies](#mapping-requests-to-endpoints-to-policies)
-  - [Wildcards](#wildcards)
-  - [Policy Semantics](#policy-semantics)
-- [Managing Policies](#managing-policies)
-  - [So You Want to Create Policies](#so-you-want-to-create-policies)
-  - [Default Policies](#default-policies)
-  - [Policy API](#policy-api)
-  - [Introspection (How to Query Permissions)](#introspection-how-to-query-permissions)
-- [Introducing Projects](#introducing-projects)
-- [Spinning up the GRPC Service Locally](#spinning-up-the-grpc-service-locally)
-  - [Start the service](#start-the-service)
-  - [Connect to the service directly](#connect-to-the-service-directly)
-  - [Connect to the service through Automate Gateway](#connect-to-the-service-through-automate-gateway)
-- [Spinning up the GRPC Service in Habitat Studio Locally](#spinning-up-the-grpc-service-in-habitat-studio-locally)
-  - [Start the service in Habitat Studio](#start-the-service-in-habitat-studio)
-  - [Connect to the service directly in Habitat Studio](#connect-to-the-service-directly-in-habitat-studio)
-  - [Connect to the service through Automate Gateway in Habitat Studio](#connect-to-the-service-through-automate-gateway-in-habitat-studio)
-- [API](#api)
-  - [func GetVersion](#func-getversion)
-  - [func IsAuthorized](#func-isauthorized)
-  - [type IsAuthorizedReq](#type-isauthorizedreq)
-  - [type IsAuthorizedResp](#type-isauthorizedresp)
-  - [type VersionInfo](#type-versioninfo)
-- [Other Resources](#other-resources)
-
-<!-- /TOC -->
+<!-- TOC depthFrom:2 depthTo:3 -->autoauto- [AuthZ Approach](#authz-approach)auto    - [Terminology](#terminology)auto    - [Introspection (How to Query Permissions)](#introspection-how-to-query-permissions)auto- [Introducing Projects](#introducing-projects)auto- [Data Layer](#data-layer)auto- [Other Resources](#other-resources)autoauto<!-- /TOC -->
```

Comment:
this is gonna mess up the table of contents formatting. supposedly the Markdown TOC plugin will auto-generate it but i haven't gotten that plugin to work.. @msorens might have guidance

Reply:
Given the amount of deleted material, I am fine to just delete the TOC in this file now. 😁

Reply:
@bcmdarroch Markdown TOC actually did the above :S Not sure why. I'll drop the table for now.